src/Autosuggest.js

```diff
@@ -208,13 +237,19 @@ export default class Autosuggest extends Component {
 
   onInputBlur() {
     this.setSuggestionsState(null);
+    this.onSuggestionUnfocused();
+    lastFocusedSuggestion = null;
```

Comment:
Why do we need to set `lastFocusedSuggestion` to `null` here? I tried to remove this line and all the tests still pass. Do we have a test that is missing, or this line is redundant?